config/rbac/role.yaml

```diff
@@ -8,29 +8,46 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - namespaces
+  - configmaps
   verbs:
   - create
   - delete
   - get
   - list
+  - update
   - watch
 - apiGroups:
   - ""
   resources:
-  - pods
+  - deployments
   verbs:
+  - create
   - delete
-  - deletecollection
   - get
   - list
+  - update
   - watch
 - apiGroups:
   - ""
   resources:
-  - pods/log
+  - namespaces
   verbs:
+  - create
+  - delete
   - get
+  - list
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - delete
```

Comment:
Is it correct that there is no 'create' permission for pods

Reply:
Yes; we never create pods directly - always through either a deployment or daemonset (or, prior to this PR being merged, a job); we do get the injector pod to ensure it's in Ready status, and the node controller has a code path that will delete pods (as a collection) to roll the kmsd daemonset.